content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+All possible deletions and insertions were applied to peptides of lengths other that 9, and the best predictive score of these was taken as optimal.
+NetMHCpan [@doi:10.1007/s00251-008-0341-z @doi:10.1186/s13073-016-0288-x] used a pseudo-sequence representation of the MHC class I molecule which included only polymorphic peptide contact residues.
+The sequences of the peptide and MHC were then represented using both sparse vector encoding and Blosum encoding, in which amino acids are encoded by matrix score vectors.
+A shallow neural network was trained on the encoded peptide sequence and MHC pseudosequence.
+Because of NetMHCpan's unique selection of an MHC amino acid sequence, it is the most suitable method covered here to make peptide affinity predictions for uncharacterized or poorly characterized MHC.
+In recent updates [@doi:10.1186/s13073-016-0288-x], NetMHCpan has been expanded to make use of the gapped sequence alignment characteristic of NetMHC, [@doi:10.1093/bioinformatics/btv639] which allows NetMHCpan-3.0 to handle both many alleles and many peptide lengths.
+MHCflurry [@doi:10.1101/054775] transforms peptide sequences using a NetMHC-type length adjustment before encoding each amino acid using a corresponding learned vector representation.
+In training the MHCflurry feed-forward neural network, the authors impute missing MHC-peptide binding affinities using a Gibbs sampling method, showing that imputation improves performance for data-sets with roughly 100 or fewer training examples.
+MHCflurry's imputation method increases its performance on poorly characterized alleles, making it competitive with NetMHCpan for this task.
+
+Using existing methods to represent uniform-length peptides, deep and mixed approaches have also shown promise towards prediction.
+Kuksa et al. [@doi:10.1093/bioinformatics/btv371] pre-trained a higher-order neural network (HONN) using a semi-restricted Boltzmann machine and compared performance with a deep neural network (DNN).
+They also propose a higher-order SVM, which, when paired with the HONN showed superior performance, which exceeded that of the DNN.
+The authors use Blosum encoding for 9-mer peptides.
+Vang et al. [@doi:10.1093/bioinformatics/btx264] combined deep learning with tools from natural language processing to approach 9-mer peptides.
+They employ a technique called distributed representation to map amino acids onto a 15-dimensional vector space based on their context relation to other amino acids.
+A convolutional neural network, titled HLA-CNN, takes peptide sequences and replaces individual amino acids with their corresponding vectors to predict binding.
+
+Deep learning's unique flexibility was recently leveraged by Bhattacharya et al., who used a gated recurrent neural network method called MHCnuggets to overcome the difficulty of multiple length peptides.
```

Comment:
Can use `gated RNN` instead of `gated recurrent neural network method`.

Is there a reference for this work?